README.md

```diff
@@ -51,7 +51,7 @@ The `@track()` decorator will expose a `tracking` prop on the component it wraps
 }
 ```
 
-The `useTracking` hook returns an object with this same shape.
+The `useTracking` hook returns an object with this same shape, plus a `Track` wrapper that can be used to pass contextual data to child components (more information about this can be found [below](https://github.com/nytimes/react-tracking#usage-with-react-hooks)).
```

Comment:
```suggestion
The `useTracking` hook returns an object with this same shape, plus a `<Track />` component that you use to wrap your returned markup to pass contextual data to child components (more information about this can be found [below](https://github.com/nytimes/react-tracking#usage-with-react-hooks)).
```